src/50_Samples_%26_Templates/Checkout_Flow.html

```diff
@@ -0,0 +1,416 @@
+            </section>
+
+            <!-- Not logged in users see a simple form for entering the shipping address. We use the `autocomplete` attributes
+              to enable [auto fill for addresses](https://developers.google.com/web/updates/2015/06/checkout-faster-with-autofill) which greatly simplifies form fill-out for users.  -->
+            <section [class]="checkoutSuccess ? 'hide' : 'checkout-section'" class="checkout-section" amp-access="NOT loggedIn">
+              <h3 class="">Enter Shipping Address</h3>
+              <label for="frmAddressS">Address</label>
+              <input name="ship-address" id="frmAddressS" placeholder="123 Any Street" autocomplete="shipping street-address">
+
+              <label for="frmCityS">City</label>
+              <input name="ship-city" id="frmCityS" placeholder="New York" autocomplete="shipping locality">
+
+              <label for="frmStateS">State</label>
+              <input name="ship-state" id="frmStateS" placeholder="NY" autocomplete="shipping region">
+
+              <label for="frmZipS">Zip</label>
+              <input name="ship-zip" id="frmZipS" placeholder="10011" autocomplete="shipping postal-code">
+
+              <label for="frmCountryS">Country</label>
+              <input name="ship-country" id="frmCountryS" placeholder="USA" autocomplete="shipping country">
+
+              <label for="shippingAddressCheck">Use Shipping as Billing Address</label>
+              <input type="checkbox" checked on="change:shippingAddress.toggleVisibility">
+            </section>
+
+            <!-- The shipping address form is optional. We hide it initially using the `hidden` attribute so that it can be toggled
+              via the `toggleVisibility` action (`on="change:shippingAddress.toggleVisibility"`).
+            -->
+            <section [class]="checkoutSuccess ? 'hide' : 'checkout-section'" class="checkout-section" hidden id="shippingAddress">
+              <h3 class="">Enter Billing Address</h3>
+              <label for="frmAddressS">Address</label>
+              <input name="ship-address" id="frmAddressS" placeholder="123 Any Street" autocomplete="shipping street-address">
+
+              <label for="frmCityS">City</label>
+              <input name="ship-city" id="frmCityS" placeholder="New York" autocomplete="shipping locality">
+
+              <label for="frmStateS">State</label>
+              <input name="ship-state" id="frmStateS" placeholder="NY" autocomplete="shipping region">
+
+              <label for="frmZipS">Zip</label>
+              <input name="ship-zip" id="frmZipS" placeholder="10011" autocomplete="shipping postal-code">
+
+              <label for="frmCountryS">Country</label>
+              <input name="ship-country" id="frmCountryS" placeholder="USA" autocomplete="shipping country">
+            </section>
+
+            <!-- ## Payment Details -->
```

Comment:
similiar -> similar

Reply:
one of the words I never get right...done